smoketest/scripts/cli/test_interfaces_bridge.py

```diff
@@ -78,6 +82,95 @@ def test_add_remove_member(self):
             self.session.delete(self._base_path + [interface, 'member'])
 
         self.session.commit()
+    
+    def test_vlan_filter(self):
+        """ Add member interface to bridge and set VLAN filter """
+        for interface in self._interfaces:
+            base = self._base_path + [interface]
+            self.session.set(base + ['address', '192.0.2.1/24'])
+            self.session.set(base + ['vif', '2','address','192.0.3.1/24'])
+
+            vlan_id = 101
+            # assign members to bridge interface
+            for member in self._members:
+                base_member = base + ['member', 'interface', member]
+                self.session.set(base_member + ['allowed-vlan', '2'])
+                self.session.set(base_member + ['allowed-vlan', '4-9'])
+                self.session.set(base_member + ['native-vlan', str(vlan_id)])
+                vlan_id += 1
+
+        # commit config
+        self.session.commit()
+        
+        # Detect the vlan filter function
+        for interface in self._interfaces:
+            with open(f'/sys/class/net/{interface}/bridge/vlan_filtering', 'r') as f:
+                flags = f.read()
+                self.assertEqual(int(flags), 1)
+        
+        # Execute the program to obtain status information
+        
+        bridge_status = Popen('bridge -j vlan show', stdin=PIPE, stdout=PIPE, shell=True)
```

Comment:
Please re-use the `vyos.util.cmd()` wrapper for this and not re-implement reading from stdout.